card-set/src/App.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import logo from './logo.svg';
+import './App.css';
+import UserCard from './card-component'
+
+function App() {
+  
+  return (
+    <div className="cards">
+      <UserCard userId="4"/>
```

Comment:
One thing you could do here is store an array of `userIds` to map over to display `UserCards`. That might make this a little bit easier to maintain, especially if we had hundreds of users. 

```js
const userIds = [4, 8, 1, 10, 12, 6];


return (
  <div className='cards' >
        { userIds.map((userId) => <UserCard key={userId} userId={userId} /> }
  </div>
);